reactive-modules/ratpack-rxjava/src/main/java/com/baeldung/rxjava/service/MovieService.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.rxjava.service;
+
+import com.baeldung.model.Movie;
+
+import rx.Observable;
+
+public interface MovieService {
```

Comment:
Since the other is called `MoviePromiseService`, should this one be called `MovieObservableService` for clarity?